src/app/personside/visittkort-v2/body/VisittkortBody.test.tsx

```diff
@@ -3,11 +3,17 @@ import * as renderer from 'react-test-renderer';
 import VisittkortBody from './VisittkortBody';
 import TestProvider from '../../../../test/Testprovider';
 import { aremark } from '../../../../mock/persondata/aremark';
+import { InformasjonElement } from '../PersondataDomain';
 
 test('viser info om bruker i visittkortbody', () => {
     const visittkortbody = renderer.create(
         <TestProvider>
-            <VisittkortBody person={aremark} />
+            <VisittkortBody
+                persondata={{
+                    feilendeSystemer: [InformasjonElement.EGEN_ANSATT],
+                    person: aremark
+                }}
+            />
         </TestProvider>
     );
```

Comment:
Burde vi hatt dette i to separate tester? Én som bare viser "vanlige" aremark, og én som tester feilede systemer?

Reply:
Mulig jeg ser feil, men burde det ikke vært en endring i `VisittkortBody.test.tsx.snap` etter denne endringen på testen?

Reply:
Bra spotta, den skulle aldri hatt `InformasjonsElement.EGEN_ANSATT` siden den kun brukes i header, så skal fjerne den nå. Det er også er årsaken til hvorfor det ikke er noen forskjell på snapshot-testen